project-koku/koku
# Adding report processing statistics to provider API response

**Overview**
`/api/v1/providers/` now has a `stats` dictionary response to give the API user some visibility as to when the last time their reports have been processed.  This will likely be one of the first debug points when a user is not seeing UI updates.

**Note**
This these stats are only available for the customer (tenant) owner.  In the future we could create an admin super user to give the cluster administrator visibility of the user statistics.  Until that is done we can only ask the user to gather these stats and deliver them to us.

**Testing**
1.  Add OCP and AWS providers and ingest multiple months of data.  Verify stats in response
2. Select a few providers from step 1 and ensure that the stats are returned successfully

**Test Results**
[koku_561_ut.txt](https://github.com/project-koku/koku/files/2738368/koku_561_ut.txt)

Closes #561
